db/migrate/20160406142950_fix_versions_item_type_for_steps.rb

```diff
@@ -0,0 +1,9 @@
+class FixVersionsItemTypeForSteps < ActiveRecord::Migration
+  def up
+    execute "UPDATE versions SET item_type = 'Steps::Approval' WHERE item_type = 'Approval'"
+  end
+
+  def down
+    execute "UPDATE versions SET item_type = 'Approval' WHERE item_type = 'Steps::Approval'"
```

Comment:
maybe it would be better to not have a `down` at all. Under what conditions would we ever want to roll this change back to a broken state?